### PR TITLE
Add QID checksum validator

### PR DESCRIPTION
**Pull request by AdamHawtin**
## Motivation and Context
We want a way of quickly validating QID's

## What has Changed
* Add QID checksum script
* Update README
* Update aliases
* Update K8s manifests

## How to Test
Run the checksum validator script with valid and invalid QID's, it should report their validity correctly and tell you the correct checksum for invalid QID's.

## Links
https://trello.com/c/PLoxk1Ud/1284-add-qid-checksum-tool-to-toolbox
https://github.com/ONSdigital/census-rm-kubernetes/pull/149

### CODE REVIEW

**Comment by AdamHawtin** [comment]
[Add QID checksum tool to toolbox (3?)](https://trello.com/c/PLoxk1Ud/1284-add-qid-checksum-tool-to-toolbox-3)

**Review by NickGrantX — APPROVED** [review]
Looks good to me 👍

**Review by rjweeks — APPROVED** [review]

File: aliases.sh [hunk 1]
```diff
@@ -16,6 +16,7 @@ alias findbad='curl -s http://$EXCEPTIONMANAGER_HOST:$EXCEPTIONMANAGER_PORT/badm
 alias viewskipped='curl -s http://$EXCEPTIONMANAGER_HOST:$EXCEPTIONMANAGER_PORT/skippedmessages | jq'
 alias resetexceptionmanager='curl -s http://$EXCEPTIONMANAGER_HOST:$EXCEPTIONMANAGER_PORT/reset'
 alias msgwizard='python bad_message_wizard.py'
+alias qidcheck='python qid_checksum_validator.py'
```
Comment (ryangrundy7) [inline aliases.sh:19]: Can you add this to the helpme alias and splashscreen?

**Review by ryangrundy7 — APPROVED** [review]
LGTM 👍

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

**Pull request merged**